### ISSUE TITLE
[Bug]: //chunk & //regen using incorrect seed.

### ISSUE THREAD

**Issue by crumpkatrina** [Multiverse/Multiverse-Core#3035]
### `/mv version -p` output

No link generated due to 502 at api.paste.gg

### Server logs

On private request.

### Server Version

```shell
1.20.1 Paper Build 195
```

### Bug Description

When selecting a //chunk and //regen 'ing it, the area is not correspondent to the world seed at all.

### Steps to reproduce

1. Load server with Paper, WorldEdit, WorldGuard versions, as well as latest Multiverse: https://github.com/Multiverse/Multiverse-Core/releases/tag/4.3.12 (no other plugins on the environment at all).
2. Use command /mv create testworld normal
3. Verify seed in game matches the seed in Multiverse config using /seed command
4. Go to any random chunk and select it with //chunk
5. Use //regen and notice it regenerates a completely different area.

Please see: https://github.com/EngineHub/WorldEdit/issues/2398

### Agreements

- [X] I have searched for and ensured there isn't already an open or resolved issue(s) regarding this.
- [X] I was able to reproduce my issue on a freshly setup and up-to-date server with the latest version of Multiverse plugins with no other plugins and with no kinds of other server or client mods.

**Comment by crumpkatrina** [comment]
It looks like //chunk & //regen is working fine when using on a world generated by Paper.
However, when the world is generated by multiverse, the //chunk & //regen command is not working correctly.

**Comment by crumpkatrina** [comment]
Paste link is as follows: https://paste.gg/p/anonymous/7228ed7dc8794226b578c18a769d24af
The commands are being run in the console to generate new worlds, that's it.

**Comment by crumpkatrina** [comment]
That is incorrect, the //regen command does not grab the seed from the server.properties command.
If you have nothing valuable to add to the issue, there is no need responding to it.

**Comment by CloudeLecaw** [comment]
> That is incorrect, the //regen command does not grab the seed from the server.properties file. If you have nothing valuable to add to the issue, there is no need responding to it.

but it should use it fron the dat file of the world folder ;)  and /seed is not working for the multi verse worlds it only works if your using the vanilla 3 dimension without a multiverse plugin.

Well at least. Worldedit grabs the seed for regen that is set in your world.dat
you can view the with a tool like NTB Explorer.
This seed should match the seed in your multiverse world if the world was generated multiverse

If //regen does not restpect this and generate something else. its not up to multiverse to handle worldedits behave.

**Comment by benwoo1110** [comment]
The seed is managed by the server, multiverse simply uses the WorldCreator api by bukkit/paper. So of world.getSeed is returning the wrong thing, i doubt its a mulitverse issue

**Comment by andrewkm** [comment]
We are having the same problem.
WorldEdit stated this is an issue with Multiverse: https://github.com/EngineHub/WorldEdit/issues/2398

**Comment by dexitma** [comment]
The problem is not only with //regen, players on my server complain that their villagers are flying or somehow running out of confinement when the player returns home. I asked one of the players at what height exactly the villager was flying, the player showed me this height, I did //regen and in this place a desert from another seed was generated. For some reason when unloading and loading a chunk, the villagers somehow use another seed to determine where they need to be or something (probably it spawns them at the highest point of the seed, which is not even there), I do not know how it works. I hope you understand what I wanted to explain

**Comment by benwoo1110** [comment]
As per above discussion, this does not seem like a multiverse issue. Open a new issue if you are facing similar problems with seed.
